dmn-decision-table/src/com/axonivy/ivy/process/element/rule/ui/DecisionTableEditor.java

```diff
@@ -226,9 +226,10 @@ private static IVariable[] getSampleScriptContext() {
     return new IVariable[] {};
   }
 
-  public void setProject(IIvyProject ivyProject) {
-    scriptEngine = ivyProject.getIvyScriptEngine();
-    project = ivyProject.project();
+  public void setProject(Project project) {
+    this.project = project;
+    var data = IDataClassManager.instance().getProjectDataModelFor(project);
```

Comment:
Faster way would be: `IProjectDataClassManager.of(project)`
Or even: `IvyScriptEngineMapper.from(project)`
But there we have to align us first with the naming!

Reply:
ahh nice, thanks for the hint.

... to me 'Mapper' suffix is hard to get used to. I'm always searching for "Adapters" first :relaxed:

Reply:
Yes may be, but Adapters are mostly empty impls in our context by now ....

Reply:
Other idea would be just `ToIvyScriptEngine.from(project)`